src/app/containers/ConsentBanner/index.canonical.jsx

```diff
@@ -21,6 +22,9 @@ const Canonical = () => {
   const onCookieAccept = () => {
     cookieOnAllow();
     updateCookiePolicy();
+    if (onDismissFocusRef) {
+      onDismissFocusRef.current.focus();
+    }
```

Comment:
- Can we change the name of this reference to the element that it is referencing? i.e. `brandRef`.
- By calling focus on this element, we could be changing a users focus. Has this been taken into consideration? A common pattern is to store the currently focussed element before stealing focus, so you can set it back afterwards.

Reply:
> Can we change the name of this reference to the element that it is referencing? i.e. brandRef.

Tbh I put a similar comment to rename this `onDismissFocusRef`, my justification was that this decouples this from the `Brand` component so if we chose to focus on something else on dismiss we would not have to change this component following the open-closed principle.

> By calling focus on this element, we could be changing a users focus. Has this been taken into consideration? A common pattern is to store the currently focussed element before stealing focus, so you can set it back afterwards.

We tried to do exactly that in this PR https://github.com/bbc/psammead/pull/4372 but we abandoned the approach as not all elements are focusable i.e. if a user was reading a paragraph we would be unable to focus them back on that paragraph; returning to the brand was a compromise we were confident would work.

Reply:
Can understand the reasoning behind both of these suggestions. In my opinion though, `onDismissFocusRef` works well here. In Header/index.jsx, the consent banner is used like this: `<ConsentBanner onDismissFocusRef={brandFocusRef} />` and I think it reads well, you've got the action and the component in there. Perhaps here we could change the `brandFocusRef` to `brandRef`?